packages/website-router/titles-tags/title-seo.ts

```diff
@@ -0,0 +1,19 @@
+export interface TitleConfig {
+  titleSEO: string;
+  URL: string;
+}
+
+export class TitleHandler implements HTMLRewriterElementContentHandlers {
+  constructor(private titleObjects: TitleConfig[]) {}
+
+  element(element: Element) {
+    if (element.tagName === 'title') {
+      const currentURL = new URL(window.location.href).pathname;
```

Comment:
You don't have `window.location.href`. It's a Worker environment. You should get from the part in code that calls `TitleHandler`